plugin/storage/integration/grpc_test.go

```diff
@@ -97,9 +179,35 @@ func TestGRPCStorage(t *testing.T) {
 	if configPath == "" {
 		t.Log("PLUGIN_CONFIG_PATH env var not set")
 	}
+
+	flags := []string{
+		"--grpc-storage-plugin.binary", binaryPath,
+		"--grpc-storage-plugin.log-level", "debug",
+	}
+	if configPath != "" {
```

Comment:
this check seems unnecessary, will it fail if we append `--grpc-storage-plugin.configuration-file` with empty string value?